inc/admin.php

```diff
@@ -509,3 +515,35 @@ function clean_orphaned_posts( $option, $post_type ) {
 
 	return true;
 }
+
+/**
+ * Check if ads.txt file already exists in the server
+ *
+ * @return void
+ */
+function adstxts_check_for_existing_file() {
+	$home_url_parsed = wp_parse_url( home_url() );
+	if ( empty( $home_url_parsed['path'] ) ) {
+		$response   = wp_remote_request( home_url( '/ads.txt' ) );
```

Comment:
We'll need something here (and in the JS) to determine whether the check should be for ads.txt or app-ads.txt .

You can probably include something in the `admin_enqueue_scripts()` function within this file based on the hook -- whether it contains `app-adstxt`  or not.

To be secure you should send it in the form `is_appstxt: true|false` and modify the URL check based on the true false statement. Don't use the variable `home_url( $passed_value )` as someone could modify that to a random file name, such as `.env` or similar.